app/templates/orders/orders_packages.hbs

```diff
@@ -40,14 +40,23 @@
         </div>
       </div>
     {{else}}
-      {{#link-to 'items.detail' ordersPackage.item.id }}
+      {{#if ordersPackage.item.isSingletonItem}}
```

Comment:
shame to have to duplicate the nested template just to accomodate the if/else clause. Would the 'disabled' attribute of link-to (https://emberjs.com/api/classes/Ember.Templates.helpers.html#method_link-to) work here or does that disabled everything inside the component? What about setting the route to '' if quantity package or creating a small partial template to reduce duplication?